app/build.gradle

```diff
@@ -80,6 +80,8 @@ dependencies {
     implementation "com.github.Raizlabs.DBFlow:dbflow-core:$rootProject.dbflowVersion"
     implementation "com.github.Raizlabs.DBFlow:dbflow:$rootProject.dbflowVersion"
 
+    implementation 'de.hdodenhof:circleimageview:3.0.1'
```

Comment:
I think it would be much better if we put the version number of CircleImageview like other dependencies at `bild.graldle project` file.